tests/test_4_logout_user.py

```diff
@@ -1,177 +1,18 @@
-    )
-    mobile_number_textbox.send_keys(user_data["phone"])
-
-    register_button = driver.find_element(
-        By.XPATH, '//button[@data-qa="create-account"]'
-    )
-    register_button.click()
-
-    # Verify that registration form is complete
-    account_created_text_present = element_present(
-        driver,
-        By.XPATH,
-        '//h2[@data-qa="account-created"]/b[contains(text(), "Account Created!")]',
-    )
-    assert account_created_text_present == True
-
-    # find "continue" button and click it
-    continue_button = driver.find_element(By.XPATH, '//a[@data-qa="continue-button"]')
-    continue_button.click()
-
-    # log out the user
-    logout_button = driver.find_element(By.XPATH, '//a[@href="/logout"]')
-    logout_button.click()
+from ae_website.AEHomePage import HomePage
+from ae_website.AELoginPage import LoginPage
 
 
 def test_case(init_driver, user_data):
-    driver = init_driver
-    driver.get("http://automationexercise.com")
-
-    register_first(driver, user_data)
-
-    # verify that there is label saying "Login to your account"
-    login_text_present = element_present(
-        driver,
-        By.XPATH,
-        '//div[@class="login-form"]/h2[contains(text(), "Login to your account")]',
-    )
-    assert login_text_present == True
-
-    # find name textbox and type person's name
-    name_textbox = driver.find_element(By.XPATH, '//input[@data-qa="login-email"]')
-    name_textbox.send_keys(user_data["email"])
-
-    # find email textbox and type person's email address
-    email_textbox = driver.find_element(By.XPATH, '//input[@data-qa="login-password"]')
-    email_textbox.send_keys(user_data["password"])
-
-    # find login button and click it
-    login_button = driver.find_element(By.XPATH, '//button[@data-qa="login-button"]')
-    login_button.click()
+    homepage = HomePage(init_driver)
+    login_page = LoginPage(init_driver)
 
-    # verify that user is logged in
-    logged_in_status = driver.find_element(
-        By.XPATH, '//ul[@class="nav navbar-nav"]/li[last()]/a'
-    )
-    assert (
-        logged_in_status.text
-        == "Logged in as " + user_data["first_name"] + " " + user_data["last_name"]
-    )
+    homepage.go_to_website()
+    homepage.check_website_visible()
+    homepage.login()
 
-    # find logout button and click it
-    logout_button = driver.find_element(By.XPATH, '//a[@href="/logout"]')
-    logout_button.click()
+    login_page.login_enter_email(user_data["email"])
+    login_page.login_enter_password(user_data["password"])
+    login_page.click_login()
 
-    # verify that user is logged out and is on log in page
-    login_text_present = element_present(
-        driver,
-        By.XPATH,
-        '//div[@class="login-form"]/h2[contains(text(), "Login to your account")]',
-    )
-    assert login_text_present == True
+    homepage.check_login_status(f"{user_data['first_name']} {user_data['last_name']}")
+    homepage.logout()
```

Comment:
you don't have any assertions after logging out. you could try to refresh the page `driver.refresh()` and check if user is in fact logged out.